CHANGELOG.md

```diff
@@ -63,6 +63,8 @@ Unreleased
 -   Split `ban` into Latin and Balinese scripts. (\#214)
 -   Split `kir` into Cyrillic and Arabic. (\#216)
 -   Split Latin (`lat`) into its dialects. (\#233)
+-   Added support for python 3.9 (\236)
```

Comment:
This line should be removed. It probably came through when you resolved the merge conflicts with master?